### PR TITLE
Improvement for the agenda and calendar search in items

### PR DESCRIPTION
**Pull request by GyanP**
https://dev.sourcefabric.org/browse/SDESK-3445

### CODE REVIEW

File: client/components/UI/Form/SelectMetaTermsInput/SelectFieldPopup.jsx [hunk 1]
```diff
@@ -319,6 +319,9 @@ export class SelectFieldPopup extends React.Component {
                                 {renderList(groupsList[g], g)}
                             </li>))
                         }
+                        {this.state.filteredList.length === 0 &&
+                            <li>No Items found</li>
```
Comment (mdhaman) [inline client/components/UI/Form/SelectMetaTermsInput/SelectFieldPopup.jsx]: missing `gettext` for translations

File: client/components/UI/Form/SelectMetaTermsInput/SelectFieldPopup.jsx [hunk 2]
```diff
@@ -396,6 +399,9 @@ export class SelectFieldPopup extends React.Component {
                                 </button>
                             </li>
                         ))}
+                        {this.state.filteredList.length === 0 &&
+                            <li>No Items Found</li>
```
Comment (mdhaman) [inline client/components/UI/Form/SelectMetaTermsInput/SelectFieldPopup.jsx]: same here as well

**Comment by mdhaman** [comment]
@GyanP can you squash commits and we can merge.

**Comment by GyanP** [comment]
@mdhaman Squashed the commits.

**Pull request merged**